build.sbt

```diff
@@ -21,6 +21,9 @@ ThisBuild / githubWorkflowBuildPostamble ++= Seq(
 
 ThisBuild / githubWorkflowPublishTargetBranches := Seq()
 
+// Scala 2.10.7 seems to be causing issues with CI
+ThisBuild / githubWorkflowScalaVersions -= "2.10.7"
+
 ThisBuild / scalaVersion := "2.12.14"
 ThisBuild / crossScalaVersions := Seq("2.10.7", "2.11.12", scalaVersion.value, "2.13.6")
```

Comment:
The real problem is here. I'm not sure why this didn't consistently fail previously, but it was probably luck. Due to the way that cross-building works, you need to invert this, otherwise any time you run `sbt ++ SOME_VERSION` you're going to be adding that version to this `crossScalaVersions`, which in turn means that the ci.yml contents check will *always* fail.

You should restructure this in the following way:

```scala
ThisBuild / crossScalaVersions := Seq("2.10.7", "2.11.12", "2.13.6", "2.12.14")
ThisBuild / scalaVersion := (ThisBuild / crossScalaVersions).value.last
```

Reply:
Thanks, your changes seem to have fixed the problem but now I have a completely unrelated issue with coveralls.